### PR TITLE
Dala 4637 changes to eurodat

### PR DESCRIPTION
**Pull request by SiegfriedSobkowiak**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [ ] If this PR includes work on the frontend, at least one `@ts-nocheck` is removed. Additionally, there should not be any `@ts-nocheck` in files modified by this PR. If no `@ts-nocheck` are left: Celebrate :tada: :confetti_ball: type-safety and remove this entry. 
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [ ] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team

### CODE REVIEW

File: dataland-eurodat-client/secret_files_templates/client.env.template [hunk 1]
```diff
@@ -23,7 +23,4 @@ QUARKUS_OIDC_CREDENTIALS_JWT_SIGNATURE_ALGORITHM=RS256
 QUARKUS_OIDC_CLIENT_GRANT_TYPE=client
 QUARKUS_OIDC_CLIENT_SCOPES=openid
 QUARKUS_OIDC_CLIENT_TLS_VERIFICATION=none
-QUARKUS_REST_CLIENT_TRUST_STORE=/crt/test.jks
-QUARKUS_REST_CLIENT_TRUST_STORE_PASSWORD="${QUARKUS_REST_CLIENT_TRUST_STORE_PASSWORD}"
-QUARKUS_OIDC_CLIENT_TLS_TRUST_STORE_FILE=/crt/test.jks
-QUARKUS_OIDC_CLIENT_TLS_TRUST_STORE_PASSWORD="${QUARKUS_OIDC_CLIENT_TLS_TRUST_STORE_PASSWORD}"
+QUARKUS_LOG_LEVEL=DEBUG
```
Comment (stewal19) [inline dataland-eurodat-client/secret_files_templates/client.env.template:26]: Soll das final bleiben oder war das nur dafür, um besser zu verstehen, was nicht funktioniert hat?

File: .github/workflows/CD.yaml [hunk 2]
```diff
@@ -186,8 +186,6 @@ jobs:
           QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD: ${{ secrets.QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}}
           QUARKUS_OIDC_CLIENT_CREDENTIALS_JWT_KEY_PASSWORD: ${{ secrets.QUARKUS_OIDC_CLIENT_CREDENTIALS_JWT_KEY_PASSWORD}}
           QUARKUS_OIDC_CLIENT_CREDENTIALS_JWT_KEY_STORE_PASSWORD: ${{ secrets.QUARKUS_OIDC_CLIENT_CREDENTIALS_JWT_KEY_STORE_PASSWORD}}
-          QUARKUS_OIDC_CLIENT_TLS_TRUST_STORE_PASSWORD: ${{ secrets.QUARKUS_OIDC_CLIENT_TLS_TRUST_STORE_PASSWORD}}
```
Comment (stewal19) [inline .github/workflows/CD.yaml:189]: Ich lege mal eine subtask an, um diese variablen auch im dataland repo zu löschen

Comment (stewal19) [reply .github/workflows/CD.yaml:189]: OK, gesehen, es gibt schon eine

File: dataland-eurodat-client/secret_files_templates/client.env.template (hunk 1 above)
Comment (SiegfriedSobkowiak) [reply dataland-eurodat-client/secret_files_templates/client.env.template:26]: Nein das muss noch weg.

**Pull request merged**